app/mailers/school_mailer.rb

```diff
@@ -22,6 +22,7 @@ class SchoolMailer < ApplicationMailer
   DIY_WORDPRESS_NOTIFICATION_TEMPLATE = "e1067a2f-b027-45a6-8e51-668e170090d1"
   PARTNERED_SCHOOL_INVITE_SIT_EMAIL_TEMPLATE = "8cac177e-b094-4a00-9179-94fadde8ced0"
 
+  # This email is currently (30/09/2021) only used for manually sent chaser emails
```

Comment:
I don't really like the idea of having code comments, and I like the idea of dating them even less :P 

If it is not clear what the email is meant to do, we should rename the method sending it to make it more explicit. And I guess we could alias methods to deal with delayed jobs for a while.

Reply:
Agree, I hate comments as well - the reason I added those was that it was extremly difficult to figure out when this mailer is being used due to a large part of apparantly unused code existing only to be run manully from the console. I dated this as well so we could make the decision later on to delete this based on how stale this comment is. It makes no sense to refactor this as it is to be deleted.